external/SWXMLHash/BUILD

```diff
@@ -0,0 +1,14 @@
+load(
+    "@build_bazel_rules_swift//swift:swift.bzl",
+    "swift_library"
+)
+
+swift_library(
+    name = "SWXMLHash",
+    module_name = "SWXMLHash",
+    srcs = glob([
+      "Source/**/*.swift"
+    ], allow_empty = False),
+    copts = ["-suppress-warnings"],
+    visibility = ["//visibility:public"]
```

Comment:
Considering this is meant as an internal dependency of SourceKitten, not a public-facing one, is this the right visibility?

Reply:
This is needed to ensure that this external dependency is visible to the repository dependencies, its a quirk of Bazel

Reply:
Seeing this issue after committing this:

```terminal
/Users/maxwellelliott/Development/SourceKitten/BUILD:7:14: in swift_library rule //:SourceKittenFramework: target '@com_github_drmohundro_SWXMLHash//:SWXMLHash' is not visible from target '//:SourceKittenFramework'. Check the visibility declaration of the former target if you think the dependency is legitimate
```

Reply:
Not a big deal to leave public, but still seems odd to me. I'd rather no-one accidentally rely on a version of SWXMLHash vendored through SourceKitten.

Reply:
Would you like to namespace this then? That can be done to make it clear it belongs to SourceKitten